### PR TITLE
add a whitelist for words like scunthorpe

### PR DESCRIPTION
**Pull request by jjhoughton**

### CODE REVIEW

**Comment by jjhoughton** [comment]
Probably should do a proper sanity check on some of these words first.

**Comment by jjhoughton** [comment]
@Jivings I don't know if you want me to create a separate grunt task  for good words or not? Please let me know

File: dicts/download.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+curl http://download.openwall.net/pub/wordlists/languages/English/1-tiny/lower.gz | gunzip > en
+curl http://download.openwall.net/pub/wordlists/languages/German/1-small/lower.gz | gunzip > de
+curl http://download.openwall.net/pub/wordlists/languages/Italian/1-small/lower.gz | gunzip > it
```
Comment (jjhoughton) [inline dicts/download.sh:5]: Don't know what license these are under, will that be an issue?

Comment (jjhoughton) [inline dicts/download.sh:5]: This is what i get in the header

```
joshua@Joshuas-MBP:~/development/grunt-naughty-words$ cat dicts/en | more
#!comment: This list has been compiled by Solar Designer of Openwall Project,
#!comment: http://www.openwall.com/wordlists/
#!comment:
#!comment: Input wordlist files:
#!comment: 10001fr.equ, 366often.mis (mwords.tar.gz) -
#!comment:  Moby (tm) Words II by Grady Ward
#!comment: englex-dict.gz - Don Olivier, Harvard School of Public Health
#!comment: pocket-dic.gz - "pocket dictionary"
#!comment: special.eng.gz - "Special English", via Rick Harrison
#!comment: wordset_4k.c - originally from a passphrase generator on sci.crypt
```

**Comment by jjhoughton** [comment]
Okay I've tested it in tophat, it seems fine.

**Comment by jjhoughton** [comment]
@Jivings Could you review this please so I can tag it and update tophat.

**Comment by Jivings** [comment]
You need to bump the version in `package.json` first.

**Comment by jjhoughton** [comment]
Version bumped in 94bdbc5d4950ba3a88e7364f4289a37c9c688eb1

**Pull request merged**